spec/features/invite_user_sign_up_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "spec_helper"
 
 feature "user invites another user and they sign up" do
-  scenario "user sends invite, invitee signs up" do
+  scenario "user sends invite, invitee signs up", js: true, vcr: true do
```

Comment:
I prefer to write `scenario "user sends invite, invitee signs up", :js, :vcr do`